lib/src/widgets/board.dart

```diff
@@ -549,6 +570,8 @@ class _BoardState extends State<Chessboard> {
     final square = widget.offsetSquare(details.localPosition);
     if (square == null) return;
 
+    widget.onTappedSquare?.call(square);
```

Comment:
Calling this `onTapped` is misleading since the callback is executed on pointer down.

Reply:
Hmm yeah I wasn't sure whether to put this into pointer down or up - Now that I think about it again, if we call it `onTapped` it should probably be in pointer up instead? And we should only invoke the callback if the pointer up happened on the same square as the previous pointer down event

Reply:
Or should we keep it in `_onPointerDown` and change the callback's name instead?